vulnerability-report.md

```diff
@@ -0,0 +1,75 @@
+##Vulnerability Report
+
+###Reviewer: Wenjing Qiang
+
+###Date: May 11, 2016
+
+
+###Injection
+
+Exploiting: Tying singel sql in uploadfile field, programeError will show.
+
+Fix: Pepalce ```curs = connection.cursor() curs.execute( "insert into taskManager_file ('name','path','project_id') values ('%s','%s',%s)" % (name, upload_path, project_id))```
+     with ```taskManager_file = File( name=name, path=upload_path, project=project_id )```
+
+
+###Broken Auth
+
+In user model, there are is_superuser and is_staff fields.  Either to add is_superuser and is_staff in exclude atrribute field or just include what we needed.
```

Comment:
Okay but where on the site was there a vulnerability? Which form in particular allows this? What page? What code do you write to exploit this vulnerability?